docs/Installation/Installation.md

```diff
@@ -237,52 +237,50 @@ To build SpECTRE with Singularity you must:
 
 SpECTRE's dependencies can be installed with
 [Spack](https://github.com/spack/spack), a package manager tailored for HPC use.
-Install Spack by cloning it into `SPACK_DIR` (a directory of your choice),
-then add `SPACK_DIR/bin` to your `PATH`.
-
-For security, it is good practice to make Spack use the system's OpenSSL
-rather than allow it to install a new copy — see Spack's documentation for
-[instructions](https://spack.readthedocs.io/en/latest/getting_started.html#openssl).
-You may need to install the development version of OpenSSL:
-* On Ubuntu (16.04), run `sudo apt-get install libssl-dev`
-* On Fedora (27), run `sudo dnf install openssl-devel`
-
-Spack works well with a module environment. We recommend
-[LMod](https://github.com/TACC/Lmod), which is available on many systems:
-* On macOS, install LMod from [brew](https://brew.sh/), then source the LMod
-  shell script by adding `. /usr/local/Cellar/lmod/YOUR_VERSION_NUMBER/init/sh`
-  to your `.bash_profile`.
-* On Ubuntu, run `sudo apt-get install -y lmod` and, for Ubuntu 17.04, add
-  `. /etc/profile.d/lmod.sh` to your `.bashrc`. For Ubuntu 16.04, the correct
-  path to add is `. /usr/share/lmod/lmod/init/bash`.
-* On Arch Linux, run `yaourt -Sy lmod` and add `. /etc/profile.d/lmod.sh` to
-  your `.bashrc`,
-* On Fedora/RHEL, GNU Environment Modules comes out-of-the-box and works equally
-  well.
-* Instructions for other Linux distros are available online.
-
-To use modules with Spack, enable Spack's shell support by adding
-`. SPACK_DIR/share/spack/setup-env.sh` to your `.bash_profile` or `.bashrc`.
-
-Once you have Spack installed and configured with OpenSSL and LMod, you can
-install the SpECTRE dependencies using Spack. One way to install the
-dependencies is with a [Spack environment](https://spack.readthedocs.io/en/latest/environments.html):
+[Install Spack](https://spack.readthedocs.io/en/latest/getting_started.html) by
+cloning it into `SPACK_DIR` (a directory of your choice). For example, you can
+clone Spack into `SPECTRE_ROOT/support/spack`. Then, enable Spack's shell
```

Comment:
Cloning into the spectre source tree seems like a recipe for people to accidentally try to merge Spack into SpECTRE as part of a PR. I don't think it's good to recommend this location.